src/config/android/readManifest.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+const xml = require('xmldoc');
+
+/**
+ * @param  {String} manifestPath
+ * @return {XMLDocument} Parsed manifest's content
+ */
+module.exports = function readManifest(manifestPath) {
+  return new xml.XmlDocument(fs.readFileSync(manifestPath, 'utf8'));
```

Comment:
Can we put fs.readFileSync into next line so it aligns better with the overall climate?